arch/arc/core/regular_irq.S

```diff
@@ -111,12 +107,12 @@ SECTION_FUNC(TEXT, _rirq_exit)
 	 * If multi bits in IRQ_ACT are set, i.e. last bit != fist bit, it's
 	 * in nest interrupt
 	 */
-	lr 	r0, [_ARC_V2_AUX_IRQ_ACT]
```

Comment:
I'd suggest to move clean-up to a separate patch so that it's clear it has no functional meaning whatsoever.

Reply:
fixed wit a macro